sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -97,7 +97,8 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
 
       // CREATE DATABASE [IF NOT EXISTS] database_name [COMMENT database_comment]
       // [LOCATION path] [WITH DBPROPERTIES (key1=val1, key2=val2, ...)];
-      case Token("TOK_CREATEDATABASE", Token(databaseName, Nil) :: args) =>
```

Comment:
changes in this file aren't really necessary since the file will be deleted anyway, but for now it's OK to keep